webapp/src/components/DateRange.js

```diff
@@ -1,10 +1,20 @@
 import React from "react";
 import moment from "moment";
+import 'moment/min/locales';
 
 const DateRange = props => {
+    const userLang = navigator.language || navigator.userLanguage;
+    const language = userLang.split("-", 1);
+
     const start = moment(new Date(props.start));
     const end = moment(new Date(props.end));
 
+    start.locale(language);
+    end.locale(language);
+
+    console.log("Language: " + language);
+    moment.locale(language);
```

Comment:
this is not needed when you call locale on start and end separately

Reply:
Done:)